cmd/gomuks/main.go

```diff
@@ -31,13 +31,14 @@ import (
 
 var wantHelp, _ = flag.MakeHelpFlag()
 var wantVersion = flag.MakeFull("v", "version", "View gomuks version and quit.", "false").Bool()
+var wantListen	= flag.MakeFull("l", "listen", "Set webserver listen address", "").String()
 
 func main() {
 	hicli.InitialDeviceDisplayName = "gomuks web"
 	exhttp.AutoAllowCORS = false
 	flag.SetHelpTitles(
 		"gomuks - A Matrix client written in Go.",
-		"gomuks [-hv]",
+		"gomuks [-hvl]",
```

Comment:
not exactly sure what format we want, but I think that this is more like an argument, so I think that we should show it more like this:

```suggestion
		"gomuks [-hv] [--listen <listen_address>]",
```